conf/messages.cy

```diff
@@ -469,32 +469,63 @@ pdf.ruling.aboutDecision.text=Rwyf wedi gwneud y penderfyniad hwn o dan adran 24
+pdf.ruling.section.askForReview.requestReview=I wneud cais am adolygiad, e-bostiwch, neu ysgrifennwch at, Swyddfa'r Cyfreithiwr a Gwasanaethau Cyfreithiol CThEF (SOLS) yn uniongyrchol. Mae'r cyfeiriad e-bost a'r cyfeiriad post i'w gweld isod.
+pdf.ruling.section.askForReview.solsTeam=Gan fod y tîm SOLS yn rhan wahanol o CThEF, bydd angen yr wybodaeth ganlynol arnynt i gael hyd i'ch achos ac i ymgymryd â'r adolygiad:
+pdf.ruling.section.askForReview.solsTeam.1=eich enw, neu enw'r busnes, fel y dangosir ar frig y llythyr hwn,
+pdf.ruling.section.askForReview.solsTeam.2=pob cyfeirnod a ddefnyddir ar frig y llythyr hwn,
+pdf.ruling.section.askForReview.solsTeam.3=enw'r tîm neu'r swyddog i CThEF a enwir ar ddiwedd y llythyr hwn,
+pdf.ruling.section.askForReview.solsTeam.4=unrhyw wybodaeth newydd,
+pdf.ruling.section.askForReview.solsTeam.5=rhesymau pam rydych yn anghytuno â'r penderfyniad.
+pdf.ruling.section.askForReview.solsTeam.cannotWrite=Os oes yna reswm pam na allwch ysgrifennu at y tîm SOLS cyn pen 30 diwrnod i ddyddiad y llythyr hwn, bydd angen esgus rhesymol arnoch am yr oedi er mwyn cael estyniad. Bydd angen i chi ysgrifennu at y tîm SOLS cyn gynted â phosibl i esbonio'r oedi wrth dderbyn y cynnig o adolygiad. Os nad oes gennych esgus rhesymol, ni ellir ymgymryd â'r adolygiad. Gallwch ofyn i dribiwnlys dderbyn apêl hwyr.
+pdf.ruling.section.askForReview.solsTeam.email=reviews@hmrc.gov.uk
+pdf.ruling.section.askForReview.solsTeam.emailText=Cyfeiriad e-bost y tîm SOLS yw: <a class="govuk-link" href="mailto:{0}">{1}</a> Os byddwch yn penderfynu defnyddio e-bost i gyfathrebu, gweler yr wybodaeth bwysig ar ddiwedd y llythyr hwn. Os ydych am gysylltu â'r tîm yn ysgrifenedig, y cyfeiriad yw –
+pdf.application.footer.solicitors.office.and.legal.services.address=Swyddfa'r Cyfreithiwr a Gwasanaethau Cyfreithiol<br> Solicitor's Office and Legal Services<br> Cyllid a Thollau EF<br> HM Revenue and Customs<br> BX9 1ZT
+
+pdf.ruling.section.appealForIndependentTribunal.title=Os hoffech apelio ar dribiwnlys annibynnol
+pdf.ruling.section.appealForIndependentTribunal.appealImmediately=Os byddai'n well gennych, gallwch apelio'n uniongyrchol ar dribiwnlys annibynnol. Gallwch hefyd symud ymlaen i apêl tribiwnlys unwaith y byddwch wedi cael canlyniad yr adolygiad ac yn anghytuno â'r penderfyniad.
+pdf.ruling.section.appealForIndependentTribunal.courtsAndTribunals=Mae'n rhaid i'r cais gyrraedd Gwasanaeth Llysoedd a Thribiwnlysoedd EF cyn pen 30 diwrnod i ddyddiad y llythyr hwn, neu cyn pen 30 diwrnod i'r dyddiad ar y llythyr sy'n nodi canlyniad yr adolygiad.
+pdf.ruling.section.appealForIndependentTribunal.requestRequirements=Dylai'r cais hwn cynnwys copïau o'r canlynol:
+pdf.ruling.section.appealForIndependentTribunal.requestRequirements.1=y llythyr hwn, neu'r llythyr canlyniad yr adolygiad
```

Comment:
Apostrophe is not displayed currently. Please use the curved one for consistency in welsh. Update to `neu’r` thanks